dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/ElasticListener.java

```diff
@@ -322,7 +322,8 @@ private static Entry removeIrrelevantProperties(final Entry entry) {
             detachedBioWorkflow.setOrganization(bioWorkflow.getOrganization());
 
             // These are for table
-            detachedBioWorkflow.setTopic(bioWorkflow.getTopic());
+            // This is some weird hack to always use topicAutomatic for search table
```

Comment:
Why always automatic as opposed to the choice in `topicSelection`? 
Might be code comment if this is `TODO`